lib/phoenix_live_view/test/live_view_test.ex

```diff
@@ -225,13 +225,15 @@ defmodule Phoenix.LiveViewTest do
     {mount_opts, lv_opts} = Keyword.split(opts, [:connect_params])
 
     put_in(conn.private[:phoenix_endpoint], endpoint || raise("no @endpoint set in test case"))
+    |> Plug.Test.init_test_session(%{})
```

Comment:
What happens if the session was already set? Do we override it?

Reply:
The new content gets merged with the previous session if one already exists

Reply:
This was just to ensure that one could call `live_isolated/3` without first initializing the session manually.

If you take a look at my closed PR (#514), originally I was popping the session data here and sending it through `init_test_session/1` to handle the string conversion, but that exposed the impending BC break, as similar conversion wasn't happening in `live_render/3`.